fbchat/client.py

```diff
@@ -88,11 +88,6 @@ def __init__(self, email, password, debug=True, user_agent=None):
     def _console(self, msg):
         if self.debug: print(msg)
 
-    def _setttstamp(self):
```

Comment:
It is not a separate function there: https://github.com/Schmavery/facebook-chat-api/blob/master/utils.js#L465